### PR TITLE
docs: added Windows and Linux instalation steps in README  

### LINKED ISSUE #105: Add installation instructions for installation from releases
There should be a section in the readme on how to install the binaries for windows and Linux from the releases.

### PR DESCRIPTION
**Pull request by sai-kumar-peddireddy**
Closes  #105

## 📑 Description
* Improved current README by adding clear windows installation steps with path configuration.

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### Sample output 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,24 @@
 
 It has SRE experience codified into it's analyzers and helps to pull out the most relevent information to enrich it with AI.
 
-## Quick Start
+## Instalation
```
Comment (AlexsJones) [inline README.md:10]:
```suggestion
## Installation
```

File: README.md [hunk 2]
```diff
@@ -7,13 +7,24 @@
...
 ```
 brew tap k8sgpt-ai/k8sgpt
 brew install k8sgpt
 ```
+##### Windows 
+
+* Download the latest Windows binaries of **k8sgpt** from the [Release](https://github.com/k8sgpt-ai/k8sgpt/releases) tab based on your system architecture.
+* Extract the downloaded package to your desired location. Configure the system *path* variable with the binary location
 
+##### Verify instalation
```
Comment (AlexsJones) [inline README.md:23]:
```suggestion
##### Verify installation
```

**Review by AlexsJones — CHANGES_REQUESTED** [review]
Thanks for contributing, a few suggestions

**Review by thschue — APPROVED** [review]
lgtm

**Pull request merged**